src/sdk/auth/auth-provider-service.ts

```diff
@@ -0,0 +1,15 @@
+import { Iland } from '../iland';
+
+/**
+ * AuthProviderService class
+ */
+export class AuthProviderService {
+  /**
+   * If we have an impersonated role initialized with the SDK, we need to update all token.
+   * @param token
+   * @returns {string} the complete token.
+   */
+  static getImpersonateToken(token?: string): string {
```

Comment:
just make this a private static function within the IlandBrowserAuthProvider

Reply:
ok i'll.